src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery[] lotteries = new Lottery[3];
+        for (int i = 0; i < lotteries.length; i++) {
+            lotteries[i] = new Lottery();
```

Comment:
you still create lottery three times. You need only one, think of it like a some magic box, generator: you have one instance, and repeatedly "press button" (call method) to get the result - new ball